release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -99,6 +99,12 @@ Using the oc-mirror CLI plug-in to mirror file-based catalog Operator images in
 
 For more information, see xref:../installing/disconnected_install/installing-mirroring-disconnected.adoc#oc-mirror-oci-format_installing-mirroring-disconnected[Mirroring file-based catalog Operator images in OCI format].
 
+[id="ocp-4-12-ironic-base-image-rhel-9"]
+==== Ironic container images use RHEL 9 base image
+In earlier versions of {product-title}, Ironic container images used Red Hat Enterprise Linux (RHEL) 8 as the base image. From {product-title} 4.12, Ironic container images use RHEL 9 as the base image. The RHEL 9 base image adds support for CentOS Stream 9, Python 3.8, and Python 3.9 in Ironic components. 
+
+For more information about the Ironic provisioning service, see the xref:../installing/installing_bare_metal_ipi/ipi-install-overview.adoc[deployment of installer-provisioned clusters on bare metal].
```

Comment:
I think it should be [Deploying install-provisioned clusters on bare metal] in brackets, since that is the name of the assembly.